src/views/InvolvementProfile/components/Membership/components/MemberList/components/MemberListItem/index.js

```diff
@@ -421,9 +421,6 @@ MemberListItem.propTypes = {
     ParticipationDescription: PropTypes.string.isRequired,
     GroupAdmin: PropTypes.bool.isRequired,
     Description: PropTypes.string.isRequired,
-
-    //Following in object are never used in this file and never passed down to another component
-    //Could be removed and everything would be fine.
```

Comment:
We didn't find out why they're not being used, instead of deleting the comment let's figure it out

Reply:
@josh-peters-2357 I was not sure why they were here or being used. But I think there is a reason why they are there.

Reply:
We'll look at this in the future